packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -82,22 +82,28 @@ the keyboard or on mouse hover. Tooltip follows the
 
 ### Tooltip compared to Popover
 
-<AspectRatio ratio="16:9">
-  <iframe
-    src="https://www.loom.com/embed/f70204bd8a4f4f21a5b1ded17f4559d1"
-    frameborder="0"
-    webkitallowfullscreen="true"
-    mozallowfullscreen="true"
-    allowfullscreen={true}
-  ></iframe>
-</AspectRatio>
+<Box marginBottom="space70">
+  <AspectRatio ratio="16:9">
+    <iframe
+      src="https://www.loom.com/embed/f70204bd8a4f4f21a5b1ded17f4559d1"
+      frameborder="0"
+      webkitallowfullscreen="true"
+      mozallowfullscreen="true"
+      allowfullscreen={true}
+    ></iframe>
+  </AspectRatio>
+</Box>
 
-#### Accessibility
+### Accessibility
 
 A tooltip **must only** be placed on a **natively focusable** HTML element. Good candidates include a [Paste Button](/components/button) or a [Paste Anchor](/components/anchor) if the tip also links to a help article.
 
 **Do not** place tooltips on non-focusable elements, like an icon.
 
+If your tooltip wraps a natively focusable HTML element that includes only an icon, make sure you set the `decorative` prop to false, and
+give the icon a title. The title of the icon should describe what the icon is representing, and the tooltip provides the additional context.
```

Comment:
We could probably be clearer here. 

To provide an example, if you have a delete phone number button who's only child is a trash can icon, the title of the icon should be the accessible name of the button action. I'd want it to say "delete phone number", not "trash can".

The title of the icon should describe the action the native element will perform, not the visual representation of the icon